engine-tests/build.gradle

```diff
@@ -62,11 +62,7 @@ dependencies {
 
     implementation("org.terasology.joml-ext:joml-test:0.1.0")
 
-    implementation('org.slf4j:slf4j-api:1.7.36') {
-        because('a backend-independent Logger')
-    }
```

Comment:
@soloturn why are we removing this here?

Reply:
only saw your comment now...

>  tested, and it is indeed not even used. so remove. also reverted back jul-to-slf4j.

so you're saying we don't use slf4j for our test stuff? interesting... what's used for logging in tests instead then? :thinking: 
I thought I saw logs in some tests, but maybe I misremember?

Reply:
slf4j comes into ```engine-tests``` via ```engine``` whose version wins. a lot of tohers bring slf4j in differing version as well, like gestalt, jnbullet, nui, nui-reflect, nui-gestatl, etc etc says ```gralde -q dependencies ``` in folder ```engine-tests```, in both compile and runtime classpath.